addons/web_tour/static/src/js/running_tour_action_helper.js

```diff
@@ -114,31 +117,26 @@ var RunningTourActionHelper = core.Class.extend({
         values.$element.trigger('focusout');
         values.$element.trigger('blur');
     },
+    _calculateCenter: ($el, selector) => {
+        const center = $el.offset();
+        if (selector && selector.indexOf('iframe') !== -1) {
+            const iFrameOffset = $('iframe').offset();
+            center.left += iFrameOffset.left;
+            center.top += iFrameOffset.top;
+        }
+        center.left += $el.outerWidth() / 2;
+        center.top += $el.outerHeight() / 2;
+        return center;
+    },
```

Comment:
Right now, your option does not use `this` so that may work but in general: don't define class methods with the arrow syntax. It is equivalent to writing:
```javascript
_calculateCenter: (function ($el, selector) {
    // ...
}).bind(this),
```

which is not what you want as `this` in this case makes no sense (this is the `window` object).